### PR TITLE
Add the announcement about the release party

### PR DESCRIPTION
**Pull request by justinas**

### CODE REVIEW

File: content/posts/go-16-release-party.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+date = "2016-02-09T12:10:00+02:00"
+tags = ["vilnius", "lithuania", "golang", "community", "meetup"]
+title = "Go 1.6 release party"
+
++++
+
+### Hey Gophers !!
+
+Go 1.6 is expected to ship mid-February
+and we are happy to anounce we are joining
```
Comment (ernestas-poskus) [inline content/posts/go-16-release-party.md]: anounce - > announce

**Comment by ernestas-poskus** [comment]
@justinas I have created the event http://www.meetup.com/Vilnius-Golang/events/228751719/ and made you as a host, I think meetup.com should allow you to manage it. 

Please check and confirm that everything is fine that I will announce it (emails members an email about event).

**Comment by justinas** [comment]
Fixed stuff. The event seems OK, feel free to proceed with the announcement.

**Comment by ernestas-poskus** [comment]
@justinas post is live http://vilniusgolang.lt/ and Meetup announced 

Thank you for contribution :+1: :bow:

**Pull request merged**